source/ShopPanel.cpp

```diff
@@ -433,6 +433,9 @@ bool ShopPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 	}
 	else if(key == SDLK_TAB)
 		activePane = (activePane == ShopPane::Main ? ShopPane::Sidebar : ShopPane::Main);
+	else if(key == 'f')
+		GetUI()->Push(new Dialog(
+			this, &ShopPanel::DoFind, "Search for:"));
```

Comment:
I see that the map find adds a new line here, but that's also a super old line of code; it's not really a part of our current conventions to add a new line unless it's for aesthetic purposes (which this doesn't seem to be to me) or because the line is too long (which this isn't).
```suggestion
		GetUI()->Push(new Dialog(this, &ShopPanel::DoFind, "Search for:"));
```